test/server_test.py

```diff
@@ -226,7 +227,7 @@ def started_imageio(tmpdir, drop_privileges="true"):
             with http.ControlClient(cfg) as c:
                 r = c.get("/tickets/no-such-ticket")
                 r.read()
-                assert r.status == 404
+                assert r.status == http_client.NOT_FOUND
```

Comment:
This change is not related to supporting initial ticket, please split to another pr.

Reply:
It is awful checking 404 or 200, even when those values are generally well-known, when we have constants that tell us exactly what are we expecting.

Since I am adding more of these in my test, I thought would be nice to have them already.

Reply:
You can use the constant in new tests. Improving unrelated tests to use constants is not required for this change, and mixing unrelated changes is bad.